tests/toolkit/ContainerTestCase.php

```diff
@@ -1,94 +0,0 @@
-<?php declare(strict_types = 1);
-
-namespace Tests\Toolkit\Contributte\CzechPost;
-
-use Nette\DI\Compiler;
-use Nette\DI\Container;
-use Nette\DI\ContainerLoader;
-use Nette\DI\Extensions\ExtensionsExtension;
-use PHPUnit\Framework\TestCase;
-
-abstract class ContainerTestCase extends TestCase
```

Comment:
Proc jsi to nenechal jak abstraktni tridu?

Reply:
ja to myslíš?

Reply:
No, proc jsi to nenechal jako abstraktni tridu, je to tu smazane.

Reply:
neviděl jsem smysl ji tam mít

Reply:
Pojdme ji tam prosim vratit. Diky :-)